addons/im_livechat/static/src/js/im_livechat.js

```diff
@@ -343,7 +343,13 @@ var Feedback = Widget.extend({
         };
         return session.rpc('/im_livechat/feedback', args).then(function () {
             if (options.close) {
-                var content = _.str.sprintf(_t("Rating: :rating_%d"), self.rating);
+                var rating_to_emoji = {
```

Comment:
you could add this on top of the file:

`var RATING_TO_EMOJI = ...`

Reply:
Small up on this comment, did you handle it ?